app/uk/gov/hmrc/emcstfe/config/AppConfig.scala

```diff
@@ -41,4 +45,20 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig, configuration: Configu
   def userAllowListBaseUrl: String = s"$userAllowListService/user-allow-list"
   def allowListEnabled: Boolean = configuration.get[Boolean]("features.allowListEnabled")
   def internalAuthToken: String = configuration.get[String]("internal-auth.token")
+
+  def getFeatureSwitchValue(feature: String): Boolean = configuration.get[Boolean](feature)
+
+
+  lazy val urlEMCSApplicationService: String =
```

Comment:
These will need to be defs as the url won't change if the feature switch is enabled as it will be saved to memory